server/routes/job/read.js

```diff
@@ -886,41 +885,44 @@ router.post('/delete', [
 router.post('/executeJob', [
   body('clusterId')
   .optional({ checkFalsy: true }).isUUID(4).withMessage('Invalid cluster id'),
-  body('jobId')
+  body('id')
       .isUUID(4).withMessage('Invalid job id'),
-  body('applicationId')
+  body('application_id')
```

Comment:
Did you change all the other places where this route is being called?

Reply:
Yes